html/pfappserver/root/static.alt/src/utils/formatter.js

```diff
@@ -0,0 +1,19 @@
+import MessageFormat from 'messageformat'
+
+export default class CustomFormatter {
```

Comment:
Please add some comments to this class.

Reply:
Not my class, taken from an old issue conversation on github/monterail. I'll add a few remarks in the header.

Reply:
Added in https://github.com/inverse-inc/packetfence/pull/3619/commits/6538d0f0e13b747af9d0bc3f6e925e06f8ab932d